tests/end-to-end/ui/06-messaging.js

```diff
@@ -199,7 +199,7 @@ function messageActionsTest() {
 				it('delete the message', () => {
 					mainContent.selectAction('delete');
 					mainContent.popupFileConfirmBtn.click();
-					browser.waitForVisible('.sweet-overlay', 3000, true);
+					flexTab.sweetAlertOverlay.waitForVisible(3000, true);
```

Comment:
Yes that's a good idea, do you think i should change some more global elements, like toast, to page?